sdk/identity/identity/review/identity.api.md

```diff
@@ -77,21 +81,36 @@ export class ChainedTokenCredential implements TokenCredential {
 }
 
 // @public
-export class ClientCertificateCredential implements TokenCredential {
+export class ClientCertificateCredential extends PersistentCredential {
     constructor(tenantId: string, clientId: string, certificatePath: string, options?: ClientCertificateCredentialOptions);
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
-    }
+}
 
 // @public
-export interface ClientCertificateCredentialOptions extends TokenCredentialOptions {
+export interface ClientCertificateCredentialOptions extends PersistentCredentialOptions {
     sendCertificateChain?: boolean;
 }
 
 // @public
-export class ClientSecretCredential implements TokenCredential {
-    constructor(tenantId: string, clientId: string, clientSecret: string, options?: TokenCredentialOptions);
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
-    }
+export class ClientSecretCredential extends PersistentCredential {
+    constructor(tenantId: string, clientId: string, clientSecret: string, options?: ClientSecretCredentialOptions);
+}
+
+// @public
+export interface ClientSecretCredentialOptions extends PersistentCredentialOptions {
+}
+
+// @public
+export interface CredentialFlow {
+    getActiveAccount(): Promise<AuthenticationRecord | undefined>;
+    getToken(scopes?: string[], options?: CredentialFlowGetTokenOptions): Promise<AccessToken | null>;
```

Comment:
interesting that this returns `| null` but `getActiveAccount` returns `| undefined` - why are we using different null sentinels in the same interface?

Reply:
getToken already returns null: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/review/identity.api.md so what I can change is getActiveAccount() to return null, if that's better.

Reply:
I think this might be a good opportunity to drop null from getToken!